pkg/apisix/route.go

```diff
@@ -48,9 +48,17 @@ func newRouteClient(c *cluster) Route {
 	}
 }
 
+// Get only looks up the cache, it's not necessary to access APISIX, since all resources
+// are created by Create, which reflects the change to cache in turn, so if resource
+// is not in cache, it's not in APISIX either.
+func (r *routeClient) Get(_ context.Context, fullname string) (*v1.Route, error) {
+	return r.cluster.cache.GetRoute(fullname)
```

Comment:
If cache return nil, need to read from admin api.
Or we need a way to resync from APISIX to cache.

Reply:
Added.